backend/apiviews/author_views.py

```diff
@@ -60,3 +60,24 @@ def get_friends(self, request, pk, *args, **kwargs):
             id_List.append(list(list(i.items())[0][1].items())[0][1])
 
         return Response({"query": "friends", "Author": id_List})
+    
+    def check_friends(self, request, authorId1,authorId2, *args, **kwargs):
+    
+        id1 = protocol_removed(authorId1)
```

Comment:
Can we just change this function name to something like `remove_protocol` instead.

Reply:
I believe this function is being called in other views too (not sure which ones). So we would have to replace those ones too

Reply:
sure I can change it in another PR then